File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.java
import java.util.*;
class Solution {
    public String reverseWords(String s) {
        String[] words_list = s.trim().split("\\s+");
        StringBuilder output = new StringBuilder();
        for (int i = words_list.length - 1; i >= 0; i--){
            output.append( (i == 0) ? words_list[i] : words_list[i] + " ");
        }
        return output.toString();
    }
}

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.cs
using System;
using System.Text.RegularExpressions;

public class Solution {
    public string ReverseWords(string s) {
        string[] s_arr= Regex.Split(s.Trim(), "\\s+");
        Array.Reverse(s_arr);

        return string.Join(" ", s_arr);


    }
}